src/pages/Todo/Todo.tsx

```diff
@@ -0,0 +1,51 @@
+import { FC, useState } from "react";
+import { Container, Typography } from "@mui/material";
+import { v4 as uuidv4 } from "uuid";
+import Header from "./components/Header";
+import TodoList from "./components/TodoList";
+import { Task } from "./types";
+import { useSnackbarAlerts } from "../../hooks/useSnackbarAlerts";
+
+const Todo: FC = () => {
+  const [todoList, setToDoList] = useState<Task[]>([]);
+  const { showSuccessSnackbar } = useSnackbarAlerts();
+
+  const addTask = (taskName: string, deadline: number): void => {
```

Comment:
In typescript there is a concept called explicit types, When the initial value is 0 the type will infer the type number from this number, in your case the function dont have return so explicitly its void. 

```suggestion
  const addTask = (taskName: string, deadline: number)=> {
```